ts/Core/Navigator.ts

```diff
@@ -2555,6 +2553,36 @@ class Navigator {
         }
     }
 
+    /**
+     * Detect if the zoomed area should stick to the minimum or maximum, #14742.
```

Comment:
Is this function also checking, if the zoomed area should stick to maximum? If yes, the function name should be renamed e.g. to `shouldStickToExtreme`. If not, then change the comment.